Modules/_operator.c

```diff
@@ -474,6 +474,44 @@ _operator_iconcat_impl(PyObject *module, PyObject *a, PyObject *b)
     return PySequence_InPlaceConcat(a, b);
 }
 
+/*[clinic input]
+_operator.in_ -> bool
+
+    a: object
+    b: object
+    /
+
+Same as a in b.
+[clinic start generated code]*/
+
+static int
+_operator_in__impl(PyObject *module, PyObject *a, PyObject *b)
+/*[clinic end generated code: output=cd4eca456096fa9a input=8bd20ace23fe9808]*/
+{
+    return PySequence_Contains(b, a);
```

Comment:
This code is wrong. You must treat -1 as an error. Same remark for `_operator.not_in()`.

Something like:

```c
int res = PySequence_Contains(b, a);
if (res < 0) {
    return NULL;
}
return PyBool_FromLong(res);
```

Reply:
Oh, I didn't know! Good to know :-)